### ISSUE TITLE
Where is a better place for isCube and isOpaqueCube methods?

### ISSUE THREAD

**Issue by calclavia** [NOVA-Team/NOVA-Core#145]
They seem out of place right now in a block's collider

**Comment by anti344** [comment]
CubeComponent?
Or wait, this is used for rendering(lighting and stuff), so, StaticBlockRenderer?

**Comment by calclavia** [comment]
Yeah. I'm thinking of putting it in renderer... At the same time, rendering should not be block, item or entity specific...

**Comment by asiekierka** [comment]
But the cube information is primarily of use to block renderers.

**Comment by Caellian** [comment]
@asiekierka However, it can also be used by a mod to add a custom mechanic.
These should be stored in CubeComponent.
What if someone decides to make a block that explodes if a nearby block is not a opaque cube? These properties should be properties of a block, not it's renderer.
